asmcomp/amd64/arch.ml

```diff
@@ -41,6 +41,7 @@ type specific_operation =
   | Ifloatarithmem of float_operation * addressing_mode
                                        (* Float arith operation with memory *)
   | Ibswap of int                      (* endianness conversion *)
+  | Iclz of int                        (* Number of leading zeros *)
```

Comment:
I should probably drop it. I want to extend coverage eventually for other integer types. It's probably clearer to use separate constructs for that. `64` is wrong in this context anyway because the operation only works on tagged 63-bit integers.